docs/examples/example.ipynb

```diff
@@ -567,14 +567,311 @@
+    "from pygments.lexers import PythonLexer\n",
+    "from pygments.formatters import HtmlFormatter\n",
+    "import IPython\n",
+    "with open('my_cool_arm.py') as f:\n",
+    "    code = f.read()\n",
+    "\n",
+    "formatter = HtmlFormatter()\n",
+    "IPython.display.HTML('<style type=\"text/css\">{}</style>{}'.format(\n",
+    "    formatter.get_style_defs('.highlight'),\n",
+    "    highlight(code, PythonLexer(), formatter)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can view more component implementations in the [examples](https://github.com/viamrobotics/python-sdk/blob/main/examples/server/v1/components.py).\n",
+    "\n",
+    "### 2. Register a new model\n",
+    "Now that we've created the modular component, we must register the resource to the registry.\n",
+    "\n",
+    "To do this, we need to define a [model](https://docs.viam.com/extend/modular-resources/#models) name and a creator function in the component. A creator function is a function that can create a resource given a config and map of dependencies.\n",
+    "In the example below, `new()` is the creator function. We get the name from the config and return the resource. We can also assign class variables based on what the config has.\n",
+    "\n",
+    "```python\n",
+    "# my-python-robot/my_cool_arm.py\n",
+    "import asyncio\n",
+    "from typing import Any, ClassVar, Dict, Mapping, Optional, Self, Tuple\n",
+    "from viam.components.arm import Arm, JointPositions, KinematicsFileFormat, Pose\n",
+    "from viam.operations import run_with_operation\n",
+    "from viam.proto.app.robot import ComponentConfig\n",
+    "from viam.proto.common import ResourceName\n",
+    "from viam.resource.base import ResourceBase\n",
+    "from viam.resource.types import Model, ModelFamily\n",
+    "\n",
+    "\n",
+    "class MyCoolArm(Arm):\n",
+    "    # Subclass the Viam Arm component and implement the required functions\n",
+    "    MODEL: ClassVar[Model] = Model(ModelFamily(\"acme\", \"demo\"), \"myarm\")\n",
+    "    \n",
+    "    def __init__(self, name: str):\n",
+    "        ...\n",
+    "    \n",
+    "    @classmethod\n",
+    "    def new(cls, config: ComponentConfig, dependencies: Mapping[ResourceName, ResourceBase]) -> Self:\n",
+    "        arm = cls(config.name)\n",
+    "        return arm\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "After the component has a defined model and creator function, the component must be registered to the `Registry`. This can be done using [register_resource_creator()](https://python.viam.dev/autoapi/viam/resource/registry/index.html#viam.resource.registry.Registry.register_resource_creator) and passing a [ResourceCreatorRegistration](https://python.viam.dev/autoapi/viam/resource/registry/index.html#viam.resource.registry.ResourceCreatorRegistration) object with the creator function as a parameter.\n",
```

Comment:
Similarly, wrap these link titles in backticks (`` ` ``) to make them look like code